src/forms/previousIncome.js

```diff
@@ -50,49 +45,48 @@ const IncomeForm = function ( props ) {
 
   var time    = 'previous',
       type    = 'income',
-      client  = props.client,
-      origin  = props.props;
+      client  = props.client;
 
 
   /** Makes sure values are propagated to 'current' properties if needed */
   var ensureCurrent = function ( evnt, inputProps ) {
     
     var keyOfCurr = inputProps.name.replace( 'previous', 'current' );
     if ( !client[ keyOfCurr ] ) {
-      origin.storeComplex( evnt, { name: keyOfCurr, value: inputProps.value } );
+      props.storeComplex( evnt, { name: keyOfCurr, value: inputProps.value } );
     }
 
     // Do the usual thing too
-    origin.storeComplex( evnt, inputProps );
+    props.storeComplex( evnt, inputProps );
 
   };  // End ensureCurrent()
 
 
   var sharedProps = { client: client, type: type, time: time,
                       storeComplex: ensureCurrent };
-
+					  
```

Comment:
The additional whitespace isn't terrible, but at some point we'll have to go back and fix it.